src/components/reviews/reviews.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import SingleReview from "../single-review/single-review";
+
+const Reviews = () => {
```

Comment:
если функция только возвращает значение, можно без return
```suggestion
const Reviews = () => (
    <ul className="reviews__list">
      <SingleReview />
    </ul>
  )
```